File: Work/fizzbuzz.py
# KS, 7th, FizzBuzz
number = 1

while number <= 50:
    print(number)
    if number % 3 == 0:
        print("Fizz")
    elif number % 5 == 0:
        print("Buzz")
    elif number % 3 == 0 and number % 5 == 0:
        print("FizzBuzz")
    number += 1

File: C/practices/fizz_buzz.c
// KS, 7th, Fizz Buzz
#include <stdio.h>

int main(){

    int num = 1;

    while (num <= 50){
        if (num % 3 == 0 && num % 5 == 0){
            printf("FizzBuzz\n");
        }else if (num % 3 == 0){
            printf("Fizz\n");
        }else if (num % 5 == 0){
            printf("Buzz\n");
        }else{
            printf("%d\n",num);
        }
        num += 1;
    }

    return 0;
}